app/population/middleware/set-results.daily-population.js

```diff
@@ -0,0 +1,82 @@
+const populationService = require('../../../common/services/population')
+
+async function setResultsDailyPopulation(req, res, next) {
+  try {
+    req.transfersIn = [
```

Comment:
This looks like dummy data? I wonder if this should/could be added separately/later on when this exists?

Reply:
This will avoid it accidentally living for ever in a live environment...

Reply:
This is sample data to show what the empty screen looks like, it will be removed before merging the PR in.